operator/v1/types_network.go

```diff
@@ -272,6 +272,16 @@ type OVNKubernetesConfig struct {
 	// +kubebuilder:validation:Minimum=1
 	// +optional
 	GenevePort *uint32 `json:"genevePort,omitempty"`
+	// HybridOverlayExtensionConfig configures an additional overlay network for peers that are
+	// not using OVN.
+	// If not specified it is unused
```

Comment:
@JacobTanenbaum is this statement necessary - imo an optional field implies "not used" unless a default is set elsewhere (and therefore is documented).

Reply:
Done